src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -1100,6 +1100,7 @@
     <Field Name="RefOrOutKeyword" Type="SyntaxToken" Optional="true">
```

Comment:
Maybe file an issue to rename this field?

Reply:
Will enter an issue. It will be a technically a breaking change though

---
In reply to: [141968512](https://github.com/dotnet/roslyn/pull/22405#discussion_r141968512) [](ancestors = 141968512)

Reply:
[https://github.com/dotnet/roslyn/issues/22441](https://github.com/dotnet/roslyn/issues/22441)

---
In reply to: [141978585](https://github.com/dotnet/roslyn/pull/22405#discussion_r141978585) [](ancestors = 141978585,141968512)